src/types/types-index.ts

```diff
@@ -0,0 +1,14 @@
+interface Repository {
+    createdAt: string;
+    description: string;
+    language: string;
+    license: string;
+    name: string;
+    owner: string;
+    score: number;
+    size: number;
+    url: string;
+    watchersCount: number;
+}
+
+export default Repository;
```

Comment:
Rename this file name to index.ts.

```suggestion
export interface Repository {
    createdAt: string;
    description: string;
    language: string;
    license: string;
    name: string;
    owner: string;
    score: number;
    size: number;
    url: string;
    watchersCount: number;
}
```